src/platforms/platform.dom.js

```diff
@@ -328,16 +330,15 @@ module.exports = {
 	initialize: function() {
 		var keyframes = 'from{opacity:0.99}to{opacity:1}';
 
-		injectCSS(this,
+		injectCSS(this, [
 			// DOM rendering detection
 			// https://davidwalsh.name/detect-node-insertion
-			'@-webkit-keyframes ' + CSS_RENDER_ANIMATION + '{' + keyframes + '}' +
-			'@keyframes ' + CSS_RENDER_ANIMATION + '{' + keyframes + '}' +
+			'@keyframes ' + CSS_RENDER_ANIMATION + '{' + keyframes + '}',
 			'.' + CSS_RENDER_MONITOR + '{' +
 				'-webkit-animation:' + CSS_RENDER_ANIMATION + ' 0.001s;' +
```

Comment:
If you're dropping `-webkit-keyframes` then I imagine `-webkit-animation` could be as well

Reply:
Removed `-webkit-animation` as well.